test/unittests/execute_call_test.cpp

```diff
@@ -147,11 +147,11 @@ TEST(execute_call, call_indirect_imported_table)
 
     const Module module = parse(bin);
 
-    auto f1 = [](Instance&, span<const uint64_t>, int) { return 1; };
-    auto f2 = [](Instance&, span<const uint64_t>, int) { return 2; };
-    auto f3 = [](Instance&, span<const uint64_t>, int) { return 3; };
-    auto f4 = [](Instance&, span<const uint64_t>, int) { return 4; };
-    auto f5 = [](Instance&, span<const uint64_t>, int) { return Trap; };
+    auto f1 = [](Instance&, span<const Value>, int) { return Value{1}; };
```

Comment:
We wont have an implicit constructor anymore?

Reply:
Not from int to ExecutionResult.